<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="2">
      <widget class="QGroupBox" name="visualization_type">
       <property name="maximumSize">
        <size>
         <width>212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Вид отображения</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="bww">
          <property name="text">
           <string>Пространственные 
характеристики</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="dens_distribution">
          <property name="text">
           <string>Распределение</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="color_matrix">
          <property name="text">
           <string>Матрица расстояний</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="validity_vector">
          <property name="text">
           <string>Валидность кластеров</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="scatter_plot">
          <property name="text">
           <string>График рассеяния</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="silhouette_plot">
          <property name="text">
           <string>Силуэт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="clusters_pie">
          <property name="text">
           <string>Мощность кластеров</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="4">
      <widget class="QGroupBox" name="plot">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Отображение</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="plot_layout"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QGroupBox" name="axis">
       <property name="maximumSize">
        <size>
         <width>212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Список элементов</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="2" column="1">
         <widget class="QSpinBox" name="cluster_number">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Кластер</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QListView" name="elements_list"/>
        </item>
        <item row="0" column="0" rowspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Число
кластеров</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QLineEdit" name="cluster_amount_display">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Объединение кластеров</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QSpinBox" name="second_join_sb"/>
        </item>
        <item row="0" column="0">
         <widget class="QSpinBox" name="first_join_sb"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="join_clusters_pb">
          <property name="text">
           <string>Объединить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="validation">
       <property name="maximumSize">
        <size>
         <width>212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>Индексы валидности</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="2" column="1">
         <widget class="QLineEdit" name="silhouette_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Dunn</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="db_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Davies-Bouldin</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="dunn_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Silhouette</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3" rowspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>212</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>ПППР</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextEdit" name="assistant_te">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Расстояние между кластерами</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListView" name="distances_list"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
